lib/docker-image-tree.coffee

```diff
@@ -1,7 +1,9 @@
 _ = require 'lodash'
 docker = require './docker'
 
-saneRepoTags = (repoTags) -> if '<none>:<none>' in repoTags then [] else repoTags
```

Comment:
:P You got me. Title changed to reflect the additional stuff :) and also I'll update the commit text actually.